src/ristretto.rs

```diff
@@ -757,7 +652,13 @@ mod test {
     fn decompress_id() {
         let compressed_id = CompressedRistretto::identity();
         let id = compressed_id.decompress().unwrap();
-        assert_eq!(id.0.compress(), CompressedEdwardsY::identity());
+        let mut identity_in_coset = false;
+        for P in &id.coset4() {
```

Comment:
Maybe we should later add some syntactic sugar for determining whether something is in the coset?